tgui/packages/tgui/interfaces/MessageMonitor.tsx

```diff
@@ -0,0 +1,338 @@
+import { useBackend, useLocalState } from '../backend';
+import { Section, Stack, Input, Button, Table, NoticeBox, Box } from '../components';
+import { Window } from '../layouts';
+
+type Data = {
+  screen: number;
+  status: Boolean;
```

Comment:
Every `Boolean` here should be `BooleanLike`, because bools do not exist in the transmission